.drone.yml

```diff
@@ -196,6 +196,49 @@ volumes:
 depends_on:
   - build
 
+---
+kind: pipeline
+type: docker
+name: package-s390x
+
+platform:
+  os: linux
+  arch: amd64
+
+node:
+  arch: s390x
+
+# TODO: change rohitsakala/drone-doceker:s390x after merging https://github.com/drone-plugins/drone-docker/pull/329
```

Comment:
When will this Todo be resolved?

Reply:
I made a PR for drone docker plugin which will add support to s390x. The drone community doesn't have s390x servers to test my PR. We have already established contact between the drone community and IBM (which provides free s390x servers to open source communities). So, my best guess is it might take couple of months for my drone PR to get merged or it might not happen also.

Reply:
I saw a similar case for windows 2004 support here https://github.com/rancher/dapper/blob/master/.drone.yml#L337 which uses a private docker image and had a note of the drone plugin PR for windows 2004 support.

Reply:
At this point it sounds like we should officially fork drone related items in rancher and add in our own support for what we need instead of having multiple personal images.

@luthermonson what is your `luthermonson/drone-docker:2004` based off of?

Reply:
the luthermonson/drone-docker images have moved.. there is a home for them https://hub.docker.com/r/rancher/drone-images/tags?page=1&ordering=last_updated now. my 2004 image for drone-docker was literally done by booting a windows 2004 server in amazon and cloning their repo and building the image and pushing to my account darren then later migrated them to the rancher repo

Reply:
@rohitsakala dont you also need to drone-git? and disable the clones like i did here https://github.com/rancher/rancher/blob/release/v2.6/.drone.yml#L679-L687

Reply:
@luthermonson where is the GH repo that creates those builds so that @rohitsakala could make a PR into there as well to get his updates in his personal build into our officially published one?

Reply:
> e-git? and disable the clones like i did here https://github.com/rancher/rancher/blob/release/v2.6/.drone.yml#L679-L687

so it turns out the drone-git stuff is done already and on the hosts... you can ignore this comment

Reply:
@deniseschannon can you re-review?